tests/E2E/test/campaigns/common_scenarios/feature.js

```diff
@@ -25,9 +25,15 @@ module.exports = {
       test('should search for the created feature', () => client.searchByValue(FeatureSubMenu.search_input.replace('%SEARCHBY', 'name'), FeatureSubMenu.search_button, data.name + date_time));
       test('should select the created feature', () => client.waitForExistAndClick(FeatureSubMenu.selected_feature));
       test('should click on "Add new feature value" button', () => client.waitForExistAndClick(FeatureSubMenu.add_value_button));
-      test('should set the "Value" input', () => client.waitAndSetValue(FeatureSubMenu.value_input, data.values[1]));
-      test('should click on "Save" button', () => client.waitForExistAndClick(FeatureSubMenu.save_value_button));
-      test('should verify the appearance of the green validation', () => client.checkTextValue(CatalogPage.success_panel, '×\nSuccessful creation.'));
+      for (let i = 0; i < dataValueNumber; i++) {
```

Comment:
I expected something really simpler :)

Without providing an additional parameter to this method, you can create the variable by doing:

```js
var dataValueNumber = data.values.length;
```